server/channels/app/permissions_migrations.go

```diff
@@ -786,95 +744,82 @@ func (a *App) getAddEnvironmentSubsectionPermissions() (permissionsMap, error) {
 		model.PermissionSysconsoleWriteEnvironmentDeveloper.Id,
 	}
 
-	// Give the new subsection READ permissions to any user with READ_ENVIRONMENT
-	transformations = append(transformations, permissionTransformation{
-		On:  permissionExists(model.PermissionSysconsoleReadEnvironment.Id),
-		Add: permissionsEnvironmentRead,
-	})
-
-	// Give the new subsection WRITE permissions to any user with WRITE_ENVIRONMENT
-	transformations = append(transformations, permissionTransformation{
-		On:  permissionExists(model.PermissionSysconsoleWriteEnvironment.Id),
-		Add: permissionsEnvironmentWrite,
-	})
-
-	// Give these ancillary permissions to anyone with READ_ENVIRONMENT_ELASTICSEARCH
-	transformations = append(transformations, permissionTransformation{
-		On: permissionExists(model.PermissionSysconsoleReadEnvironmentElasticsearch.Id),
-		Add: []string{
-			model.PermissionReadElasticsearchPostIndexingJob.Id,
-			model.PermissionReadElasticsearchPostAggregationJob.Id,
+	return permissionsMap{
```

Comment:
Nit pick: I personally find long `return` statements hard to read and would prefer a local variable instead. 2/5

Reply:
I condensed it some. Let me know what you think. The return statement is still a little long but it is consistent with the rest of the file.